docs/components/workflow.md

```diff
@@ -1,4 +1,4 @@
-## Components
+# Workflow
```

Comment:
same as above

Reply:
It needs to have some config. See my comment above.